### ISSUE TITLE
Should ask for host address

### ISSUE THREAD

**Issue by striderwhite** [brockroadhouse/homestead-mksite#1]
My homestead has different IP than 192.168.56.56 - suggestion is to ask for different hostname and if not provided default to `192.168.56.56`

**Comment by brockroadhouse** [comment]
There's some settings at the top of the script you can change:
```
homestead_path="$HOME/homestead"
project_path="$HOME/projects"
homestead_ip="192.168.56.56"
git_repo="git@github.com:PDERAS"
domain_prefix=""
domain_suffix=".test"
```

**Comment by brockroadhouse** [comment]
Closed with 97ec1d906b954677555db3d2f18c3dd553ba8543

No need to change any settings for the IP address. The script will grab it from your homestead.yaml file (as long as `homestead_path` is set correctly in settings)
